src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,54 @@
 package core.basesyntax;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+
+        SimpleDateFormat dateFormat = new SimpleDateFormat("dd.MM.yyyy");
```

Comment:
1. should be `private static final` class-level constant
2. please, use `DateTimeFormatter` instead